stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2487,20 +2494,305 @@ internal class _RawNative${Self}Buffer :
     return buffer
   }()
 
+  internal init(_doNotCallMe: ()) {
+    _sanityCheckFailure("Only initialize these by calling create")
+  }
+
+#if _runtime(_ObjC)
+  func enumerator() -> _NSEnumerator {
+    return _Native${Self}NSEnumerator<${AnyTypeParameters}>(
+        _Native${Self}Storage(_buffer: self))
+  }
+
+  @objc(copyWithZone:)
+  func copy(with zone: _SwiftNSZone?) -> AnyObject {
+    return self
+  }
+
+  var count: Int { 
+    return 0 
+  }
+
+  @objc(countByEnumeratingWithState:objects:count:)
+  func countByEnumerating(
+    with state: UnsafeMutablePointer<_SwiftNSFastEnumerationState>,
+    objects: UnsafeMutablePointer<AnyObject>?, count: Int
+  ) -> Int {
```

Comment:
I think I implemented a generic version of this that works for all `Sequence`s conforming to `NSFastEnumeration` or whatever.  You might want to look around to see if you need this.